amundsen_common/models/table.py

```diff
@@ -61,26 +61,27 @@ class Meta:
 
 
 @attr.s(auto_attribs=True, kw_only=True)
-class Statistics:
+class Stat:
```

Comment:
unfortunately, this will break all the other services (https://github.com/amundsen-io/amundsenmetadatalibrary/search?q=Statistics&unscoped_q=Statistics) . I know this is not perfect. But let's keep it as it is and do it differently (e.g  add another class named Stat and mark `Statistics` deprecated etc)

Reply:
So the suggestion is to change it all over to a new `Stat` class here and only remove `Statistics` after references are gone in the other repos? Sure, that works!

- [ ] new class, not rename